projects/storefrontapp-e2e-cypress/cypress/helpers/checkout-forms.ts

```diff
@@ -28,21 +28,33 @@ export interface PaymentDetails {
 export function fillShippingAddress(shippingAddressData: ShippingAddressData) {
   cy.get('cx-address-form').within(() => {
     cy.get('[formcontrolname="titleCode"]').ngSelect('Mr.');
-    cy.get('[formcontrolname="firstName"]').type(shippingAddressData.firstName);
-    cy.get('[formcontrolname="lastName"]').type(shippingAddressData.lastName);
-    cy.get('[formcontrolname="line1"]').type(shippingAddressData.address.line1);
-    cy.get('[formcontrolname="line2"]').type(shippingAddressData.address.line2);
+    cy.get('[formcontrolname="firstName"]')
```

Comment:
I guess you clear those fields for editing.

Reply:
Yes, that's correct.